crates/plugins/src/uninstall.rs

```diff
@@ -0,0 +1,38 @@
+use crate::{get_manifest_file_name, PLUGIN_MANIFESTS_DIRECTORY_NAME};
+use anyhow::Result;
+use std::{fs, path::PathBuf};
+pub struct PluginUninstaller {
+    plugin_name: String,
+    plugins_dir: PathBuf,
+}
+
+impl PluginUninstaller {
+    pub fn new(plugin_name: &String, plugins_dir: PathBuf) -> Self {
```

Comment:
Why `&String`? This should probably be `&str`. Does `cargo clippy` catch this by any chance?

Reply:
Good catch. `cargo clippy` does not throw any error but I will make the change.